sunpy/map/tests/test_reproject_to.py

```diff
@@ -51,6 +51,7 @@ def test_reproject_to_hgs(aia171_test_map, hgs_header):
     aia171_test_map.reproject_to(hgs_header).plot()
 
 
+@figure_test
```

Comment:
I get that these are not standard figure tests but I still think they should be marked as such.